stix2/test/test_environment.py

```diff
@@ -184,3 +184,25 @@ def test_parse_malware():
     assert mal.modified == FAKE_TIME
     assert mal.labels == ['ransomware']
     assert mal.name == "Cryptolocker"
+
+
+def test_created_by():
+    identity = stix2.Identity(**IDENTITY_KWARGS)
+    factory = stix2.ObjectFactory(created_by_ref=identity.id)
+    env = stix2.Environment(store=stix2.MemoryStore(), factory=factory)
+    env.add(identity)
+
+    ind = env.create(stix2.Indicator, **INDICATOR_KWARGS)
+    creator = env.created_by(ind)
+    assert creator.id == identity.id
```

Comment:
The actual objects should be equal, not just their ID's, correct? It might help to be clear that the objects themselves are equal.